### DISCUSSION TITLE
flowershow Docker version. (Using Coolify)

### DISCUSSION THREAD

**Discussion by ARHAEEM** [flowershow/flowershow#600] [General and Help] [answered]
Hello, I'm using Coolify.io as a Docker manager for my server. I would like to know if there is an official Docker image for Flowershow? Thanks in advance.

**Comment by rufuspollock** [comment, marked as answer]
@ARHAEEM there isn't one atm. it should be pretty easy to create though 😄

**Comment by Nachtschwingeswildhunt** [comment]
I'd be interested as well, but sadly I do not know how to create such an image. Any chances of there being an official version in the future?

**Reply by rufuspollock** [reply]
We don't have plans ourselves but you can open a feature request and we'll see what happens 😄
